.github/workflows/ci_tests.yaml

```diff
@@ -83,39 +83,21 @@ jobs:
         shell: bash -l {0}
         run: conda list
 
-      # Cache the ${HOME}/.gmt directory, for docs and testing
-      - name: Cache GMT directory
-        uses: actions/cache@v2
-        id: cache
+      # Download cached remote files (artifacts) from Github
+      - name: Download remote data from Github
+        uses: dawidd6/action-download-artifact@v2
```

Comment:
I'll upgrade the action to v2.6.3, there's been some changes https://github.com/dawidd6/action-download-artifact/compare/v2.0.0...v2.6.3, and we can leave out setting the github_token.